recipe/setup.cfg.patch

```diff
@@ -0,0 +1,13 @@
+diff --git setup.cfg setup.cfg
+index 49917a6..0651f4c 100644
+--- setup.cfg
++++ setup.cfg
+@@ -15,6 +15,8 @@ addopts = -p no:warnings
+ 
+ [ah_bootstrap]
+ auto_use = True
++offline = True
```

Comment:
maybe add a reminder comment that this patch can be removed once https://github.com/pypa/pip/issues/4383 is fixed?

Reply:
Done